README.md

```diff
@@ -1 +1,68 @@
-# gleaming-dragonfly-calculator
\ No newline at end of file
+#  Mac Calculator Clone
+## Team:
+
+Frankie Rocha
+
+Project: [#150-Mac Calculator Clone](http://jsdev.learnersguild.org/goals/150-Mac_Calculator_Clone.html)
+
+Team Name: gleaming-dragonfly
+
+## Stage 1
+
+**Specs**
+
+ - [x] All text is in the `Roboto` web font
+ - [x]  Your HTML and CSS follows this style guide
+ - [x]  The calculator is positioned in the center of the page, both vertically and horizontally
+ - [x]  The calculator is a fixed size. It does not change in size when the page resizes.
+ - [ ]  If the window is too small for the calculator, the page scrolls, both vertically and horizontally
+ - [x]  Each button has a CSS transition to slightly darken the background color on hover over 100ms
+ - [x]  Each button has a CSS transition to slightly darken the background color on click over 100ms
+ - [x]  All class names re: the calculator are name-spaced under .calculator-…
```

Comment:
@deadlyicon I think prefixing everything with `calculator` seems weird, personally. But I also know little to nothing about good CSS file design so I think you explained above helped, but I also still don't know why we need to prefix everything with calculator.